setup.json

```diff
@@ -74,6 +74,7 @@
     "install_requires": [
         "aiida_core[atomic_tools]>=1.0.0b5",
         "matplotlib<3.0.0; python_version<'3'",
+        "qe-tools==1.1.2",
```

Comment:
So this commit does not depend on 1.1.3 yet?

Reply:
It should have, even though I haven't been able to release the package on PyPI since I don't have the credentials. Anyway, the PR I pushed was wrong. I have corrected it now. Just have to wait for `qe-tools==1.1.3` to be pushed.